pkg/tests/cross-tests/rapid_tv_gen.go

```diff
@@ -401,6 +402,7 @@ func (tvg *tvGen) WithNullAndUnknown(gen *rapid.Generator[tv]) *rapid.Generator[
 				nullGen := rapid.Just(tftypes.NewValue(tv0.typ, nil))
 				options = append(options, nullGen)
 			}
+			_ = rapid.Bool().Draw(t, "consumeBitstream2")
```

Comment:
Well, hmm, I bet this is not idiomatic but I don't know if this introduces any semantic problems also. 

Is the problem simply in the degenerate case where we have:

```
tvg.generateUnknowns == false
!tv0.schema.Required == false
options == []*rapid.Generator[tftypes.Value]{gen}
gen == rapid.OneOf(options...)
```

Should it stop wrapping gen with rapid.OneOf(gen) in that case maybe that fixes the problem?

Reply:
This one feels especially wrong. We do draw entropy from `t` on L393+: `tv0 := gen.Draw(t, "tv")`. The only way we can get this error message (as I understand it), is if `gen.Draw(t, "tv")` doesn't draw entropy, implying the bug is somewhere else.

Reply:
I have not verified both are needed, just that the panic no longer happens with them.